### DISCUSSION TITLE
Default units not working

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#8858] [Q&A / Support] [answered]
Hello,

I have created one object and add the Quantity value field type, Now I have set the default to the unit but it's not working.
**Set default unit:**
![image](https://user-images.githubusercontent.com/52573912/115704832-785add80-a389-11eb-9362-f827fd594e6e.png)

**Not working when add new object**
![image](https://user-images.githubusercontent.com/52573912/115705077-b7892e80-a389-11eb-8135-a7fec5617193.png)

can anyone have an idea about this, how can I set the default unit.

**Comment by NiklasBr** [comment]
It's a known issue with no fix unfortunately:  #6381

It affects both Classification Stores and regular Data Object classes

**Comment by ghost** [comment, marked as answer]
This is working fine, I found my mistake but Pimcore also needs to do some validation.

Like:
When the user going for add QuantityValue and click on **Add** button it shows the unique identifier popup and the user can add anything there this is the current functionality of pimcore.

**What should be there**  When the user clicks on **Add** button and show a unique identifier popup and the user can add only numeric value.

![image](https://user-images.githubusercontent.com/52573912/116389511-a4b7a380-a83a-11eb-89fb-f9acf553a307.png)
